src/core/ext/filters/max_age/max_age_filter.c

```diff
@@ -288,45 +284,37 @@ static grpc_error* init_channel_elem(grpc_exec_ctx* exec_ctx,
   chand->channel_stack = args->channel_stack;
   chand->max_connection_age =
       DEFAULT_MAX_CONNECTION_AGE_MS == INT_MAX
```

Comment:
2. works out however if we change the return type of add_random_max_connection_age_jitter to grpc_millis